xcape/core.py

```diff
@@ -162,3 +177,85 @@ def calc_cape(p, t, td, ps, ts, tds, source='surface', ml_depth=500., adiabat='p
     else:
         cape, cin = _reshape_outputs(cape_2d, cin_2d, shape=original_shape)
         return cape, cin
+    
+
+def calc_srh(p, t, td, u, v,  ps, ts, tds, us, vs, depth = 3000, 
+             vertical_lev='sigma', pres_lev_pos=1, output_var='all'):
+    """
+    Calculate cape for a set of profiles over the first axis of the arrays.
+
+    Parameters
+    ----------
+    p : array-like
+        Pressure in mb.
+        When vertical_lev='model', p.shape = t.shape = (nlev, x, y, ...)
+        When vertical_lev='pressure', p.shape = t.shape[0] = (nlev)
+    t : array-like
+        Temperature in Celsius
+    td : array-like
+        Dew point temperature in Celsius
+        
+    ps : array-like
+        Surface Pressure in mb.
+    ts : array-like
+        Surface Temperature in Celsius
+    tds : array-like
+        Surface Dew point temperature in Celsius
+        
+    depth : float, optional
+        Depth (m) of SRH layer.
+
+    vertical_lev : {'sigma', 'pressure'}
+        Which vertical grid is used
+    pres_lev_pos :  array-like,
+        location in fortran values (1: nlev) of where p <= ps. 
+        When vertical_lev='model', pres_lev_pos = 1
+        When vertical_lev='pressure', pres_lev_pos.shape = ps.shape
```

Comment:
I don't understand this documentation. Can you rewrite it to make it more clear?